torchvision/prototype/models/inception.py

```diff
@@ -28,11 +28,11 @@ class Inception3Weights(Weights):
     )
 
 
-def inception_v3(weights: Optional[Inception3Weights] = None, progress: bool = True, **kwargs: Any) -> Inception3:
+def inception_v3(weights: Optional[InceptionV3Weights] = None, progress: bool = True, **kwargs: Any) -> Inception3:
```

Comment:
The naming is inconsistent. The model class is called `Inception3`, while the mode builder inception_**v**3. It's unclear how we should name the weights. Following the convention from ResNets this should have been `Inception_V3` which is ugly. I propose not to spend more time on this now and tackle it at #4652. I've already added a reference for this issue.